rclpy/src/rclpy/_rclpy.c

```diff
@@ -4226,7 +4227,7 @@ _rclpy_on_time_jump(
         clock_change = "RCL_SYSTEM_TIME_NO_CHANGE";
         break;
       default:
-        PyErr_Format(PyExc_RuntimeError, "Unknown time jump type %d", time_jump->clock_change);
+        PyErr_Format(RCLError, "Unknown time jump type %d", time_jump->clock_change);
```

Comment:
This is not an error from `rcl`.
```suggestion
        PyErr_Format(PyExc_RuntimeError, "Unknown time jump type %d", time_jump->clock_change);
```